app/views/pages/get-started/contact-us-tutorial/building-the-ui.liquid

```diff
@@ -0,0 +1,99 @@
+---
+metadata:
+  title: Building the User Interface
+  description: Follow these step-by-step tutorials to build a Contact Us form on platformOS
+converter: markdown
+---
+
+To implement the _Contact Us_ form, we'll create a homepage with two fields: `email` and `body`, where users can enter their contact information. 
+
+{% include 'alert/tip', content: 'Creating a homepage is a bit special compared to creating a regular [page](/developer-guide/pages/pages). For detailed documentation, refer to the [Homepage](/developer-guide/pages/pages#homepage) documentation.' %}
+
+
+## Requirements
+
+* Followed all steps of the [Quick Setup Guide](/get-started/contact-us-tutorial/#quick-setup-guide) for building 
+* HTML knowledge
+* [Pages](/developer-guide/pages/pages)
+
+## Create a directory for pages
+
+First, create a directory for your pages within the `app` directory:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+mkdir -p app/views/pages
+</code></pre>
+
+`-p`: an option to be able to create multiple nested directories at once.
+
+{% include 'alert/tip', content: 'You can have multiple files with the same path, so organizing your files clearly is essential.' %}
+
+
+## Create a home page file
+
+We will use HTML and Liquid templating to create the form structure and input fields. Create an `index.html.liquid` file in `app/views/pages/` and add the following HTML code:
+
+#### app/views/pages/index.html.liquid
+
+{% raw %}
+```liquid
+<h2>Contact Us</h2>
+<form>
+    <div>
+        <label for="email">Email</label>
+        <input type="text" name="contact[email]" id="email">
+    </div>
+<div>
+      <textarea name="contact[body]"></textarea>
+    </div>
+    <input type="submit" value="Send">
+  </form>
+```
+{% endraw %}
+
+You might notice that we used `index.html.liquid` for our homepage file instead of `index.html`. This is because our tool can download websites and store them in HTML format, which can still be rendered by platformOS. Although using `.liquid` is not strictly necessary, we recommend it for consistency and ease of use.
```

Comment:
I am not sure about this, but we had some talks before and I believe we settled for not using `.html.liquid` but just `.liquid` instead for our tutorials and codebases. I am quite sure about this as we are using those extensions in `pos-cli init` that has been updated lately to follow those guidelines. Needs to be validated probably.

Reply:
oh so it's the other way around - for shortcut and later we recommend skipping `html`, but I believe it's good to explain this concept, so people could do index.liquid and index.json.liquid if needed